week-3/reading-list/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
```

Comment:
I see this line but don't see any `.css` file here. Was it the part of the task to style the page or not? I'm not really sure but I see one of the requirements:

> The page is visually appealing

Maybe you just missed to add it to commit?